sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubClient.cs

```diff
@@ -545,7 +546,7 @@ private static void ValidateClientOptions(EventHubClientOptions clientOptions)
 
             // A proxy is only valid when web sockets is used as the transport.
 
-            if ((clientOptions.TransportType == TransportType.AmqpTcp) && (clientOptions.Proxy != null))
+            if ((!clientOptions.TransportType.IsWebSocketTransport()) && (clientOptions.Proxy != null))
             {
                 throw new ArgumentException(String.Format(CultureInfo.CurrentCulture, Resources.ProxyMustUseWebsockets), nameof(clientOptions));
```

Comment:
Does it matter that the resources `Websockets` is not pascal like in the `IsWebSocketTransport` extension method?

Reply:
Technically, its two words `web sockets`  so the resource should match.    I'll tweak it.